src/main/resources/hudson/plugins/git/extensions/impl/FirstBuildChangelog/config.groovy

```diff
@@ -0,0 +1,7 @@
+package hudson.plugins.git.extensions.impl.FirstBuildChangelog
+
+def f = namespace(lib.FormTagLib)
+
+f.entry(field: "makeChangelog") {
+    f.checkbox(title: _("Make Changelog"))
+}

```

Comment:
Please use a jelly file to define the checkbox rather than a groovy file.  Refer to the jelly files in the repository that implement checkboxes for an example.

The [Jenkins design library](https://plugins.jenkins.io/design-library/) is structured to encourage more and more use of jelly and less and less use of groovy to define Jenkins user interface components.